logs/logs.go

```diff
@@ -0,0 +1,56 @@
+package logs
+
+import (
+	jwt "github.com/dgrijalva/jwt-go"
+	dockerClient "github.com/fsouza/go-dockerclient"
+	"github.com/rancherio/host-api/app/common/connect"
+	"github.com/rancherio/host-api/config"
+	"net/http"
+	"strconv"
+)
+
+func GetLogs(rw http.ResponseWriter, req *http.Request) error {
+	urlToken := req.URL.Query().Get("token")
+	token, err := jwt.Parse(urlToken, func(token *jwt.Token) (interface{}, error) {
+		return token, nil
+	})
+
+	exec := token.Claims["exec"].(map[string]interface{})
+	container := exec["Container"].(string)
+	follow, found := exec["Follow"].(bool)
+	if !found {
+		follow = true
+	}
+	tailTemp, found := exec["Lines"].(float64)
+	var tail string
+	if found {
+		tail = strconv.FormatFloat(tailTemp, 'f', 0, 64)
+	} else {
+		tail = "100"
+	}
+
+	client, err := dockerClient.NewClient(config.Config.DockerUrl)
+	if err != nil {
+		return err
+	}
+
+	conn, err := connect.GetConnection(rw, req)
+	if err != nil {
+		return err
+	}
+
+	logopts := dockerClient.LogsOptions{
+		Container:    container,
+		OutputStream: conn,
+		ErrorStream:  conn,
+		Follow:       follow,
+		Stdout:       true,
+		Stderr:       true,
+		Timestamps:   true,
+		Tail:         tail,
+		RawTerminal:  true,
+	}
+	go client.Logs(logopts)
```

Comment:
You are right. I was doing that during debug to send messages to the client after that spot. Will fix.